plugins/android/src/main/java/androidLibrary.kt

```diff
@@ -1,20 +1,16 @@
-import tools.forma.android.feature.AndroidLibraryFeatureConfiguration
-import tools.forma.android.feature.androidLibraryFeatureDefinition
-import tools.forma.android.feature.applyFeatures
-import tools.forma.android.feature.kotlinAndroidFeatureDefinition
-import tools.forma.android.target.LibraryTargetTemplate
+import org.gradle.api.Project
+import tools.forma.android.feature.*
```

Comment:
please avoid wildcard imports

Reply:
It is just default code style settings from forma project, it brakes sometimes even if I set my own setting before. We should share code style for project using .idea

Reply:
Of course ill fix it)

Reply:
You can commit project settings in the repo as well